module/metrics/herocache.go

```diff
@@ -29,6 +29,10 @@ func NetworkReceiveCacheMetricsFactory(registrar prometheus.Registerer) *HeroCac
 	return NewHeroCacheCollector(namespaceNetwork, ResourceNetworkingReceiveCache, registrar)
 }
 
+func PublicNetworkReceiveCacheMetricsFactory(registrar prometheus.Registerer) *HeroCacheCollector {
+	return NewHeroCacheCollector("public_"+namespaceNetwork, ResourceNetworkingReceiveCache, registrar)
```

Comment:
here as well, i think we had a var to represent this, rather than the string concat